### ISSUE TITLE
Level Music Volume Setting...

### ISSUE THREAD

**Issue by datahead8888** [Secretchronicles/TSC#486]
As discussed in IRC, we might consider allowing the volume to be set for each song for each level.

Logs from today's discussion (whole day) - http://chatlogs.secretchronicles.de/htmllogs/2015-11-10.log.html

**Comment by Quintus** [comment]
I don't think this is a good idea. Levels should have equal sound, otherwise the level designer may try to annoy the user with extra loud music.
